simulator/export/export_test.go

```diff
@@ -578,6 +694,7 @@ func TestService_Export_IgnoreErrOption(t *testing.T) {
 	}
 }
 
+// TODO: Ensure that these namespaces are actually applied. Issue #139.
```

Comment:
I haven't fixed these Import tests cuz the applying of namespace is difficult to test about `create/patch` logic using the fakesimpleclientset, currently. But we need to fix it on #139 after all. So I would do it at that time.